### PR TITLE
[RFR] Added two button fixture and automated manual test

### PR DESCRIPTION
**Pull request by spusateri**
### PRT Run

{{pytest: cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py -k "test_generic_object_button_delete_multiple" -v}}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 4c28e5ac182ca869d8f162c777755da9c51ce1db

The *local* fixture **`two_buttons_without_group`** is used in the following files:
- **cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py**
    - *test_generic_object_button_delete_multiple*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by mshriver** [comment]
PRT failing at the moment because of wharf connection timeouts.

File: cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py [hunk 1]
```diff
@@ -332,9 +350,8 @@ def test_generic_object_button_delete_flash(button_without_group):
     )
 
 
-@pytest.mark.manual
 @pytest.mark.meta(blockers=[BZ(1753281), BZ(1753388)], coverage=[1753281, 1753388])
```
Comment (mshriver) [inline cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py:353]: The `coverage` list here should be updated to `automates`, as the meta marker processes them differently for BZ reports.

Comment (mshriver) [inline cfme/tests/automate/generic_objects/buttons/test_generic_class_custom_button.py:353]:
```suggestion
@pytest.mark.meta(blockers=[BZ(1753281), BZ(1753388)], automates=[1753281, 1753388])
```

**Pull request merged**